### PR TITLE
calculate used space correctlly

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
calculate all cf's space

### CODE REVIEW

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -1054,9 +1056,15 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         stats.set_capacity(capacity);
 
         // Must get the total SST file size here.
-        let used_size = self.engine
-            .get_property_int(ROCKSDB_TOTAL_SST_FILE_SIZE_PROPERTY)
-            .expect("rocksdb is too old, missing total-sst-files-size property");
+        let mut used_size: u64 = 0;
+        for cf in DEFAULT_CFS {
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: we can use engine `cf_names()` directly.

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -41,6 +41,8 @@ use kvproto::metapb;
 use util::worker::{Worker, Scheduler};
 use util::transport::SendCh;
 use util::get_disk_stat;
+use util::rocksdb;
+use storage::DEFAULT_CFS;
```
Comment (ngaut) [inline src/raftstore/store/store.rs]: I suppose we should rename DEFAULT_CFS to ALL_CFS.

**Comment by ngaut** [comment]
LGTM

**Comment by zhangjinpeng1987** [comment]
@siddontang @BusyJay @disksing PTAL

File: src/raftstore/store/store.rs [hunk 3]
```diff
@@ -1054,9 +1056,15 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         stats.set_capacity(capacity);
 
         // Must get the total SST file size here.
-        let used_size = self.engine
-            .get_property_int(ROCKSDB_TOTAL_SST_FILE_SIZE_PROPERTY)
-            .expect("rocksdb is too old, missing total-sst-files-size property");
+        let mut used_size: u64 = 0;
+        for cf in ALL_CFS {
```
Comment (disksing) [inline src/raftstore/store/store.rs]: Use `engine.cf_names()` will be more accurate.

**Comment by disksing** [comment]
LGTM.

**Pull request merged**